locations/Bahraich/inputs/generation/grid_inputs.yaml

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue May 31 19:25:45 2022
+
+@author: pahar
+"""
+exchange_rate:25000 #[LBP/$]
+
+grid:
+
+- name: dieselg # DIESEL GENERATOR GRID
+    costs:
```

Comment:
What are the sub-grids for the diesel? I think that these should reflect the same structure as the main grid, i.e.,
```
grids:
  - name: edl
    tiers: # <<< Can change this name to something similar?
    - name: edl_threshold_1
      costs:
        etc...
    - name: edl_threshold_2
    ... etc...
    emissions:
      ... same for all grids ...
  - name: diesel_grid
    tiers:
    - diesel_5A:
      costs:
      ... specific to 5A diesel ...
    - diesel_5A:
      costs:
      ... specific to 5A diesel ...
    emissions:
    ... relevant to all, same as grid grids...